src/modules/emotes/emojis.js

```diff
@@ -82,6 +83,15 @@ class Emojis extends AbstractEmotes {
                 this.emotes.set(code, emote);
             });
     }
+
+    convertEmojis(sendState) {
+        sendState.message =  sendState.message.split(' ').map(piece => {
```

Comment:
this is gonna create a chicken & the egg problem.. emojis run before something that replaces text that needs a replacement